scripts/vote_2023_10_31.py

```diff
@@ -0,0 +1,129 @@
+2. Switch the Easy Track DAI top-up setup to the Easy Track DAI, USDT, and USDC top-up setup for all [Lido Contributors Group multisigs](https://research.lido.fi/t/ref-introducing-the-lido-contributors-group-including-pool-maintenance-labs-and-argo-technology-consulting/3069) ([RCC](https://app.safe.global/settings/setup?safe=eth:0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437), [PML](https://app.safe.global/settings/setup?safe=eth:0x17F6b2C738a63a8D3A113a228cfd0b373244633D), and [ATC](https://app.safe.global/settings/setup?safe=eth:0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956)). Items 3-8.
+
+The new version of contracts was [audited by Oxorio](LINK_TO_AUDIT).
+
+And last motion is
+
+3. **stETH transfer to the [Lido Contributor's Group multisigs](https://research.lido.fi/t/ref-introducing-the-lido-contributors-group-including-pool-maintenance-labs-and-argo-technology-consulting/3069)** ([RCC](https://app.safe.global/settings/setup?safe=eth:0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437), [PML](https://app.safe.global/settings/setup?safe=eth:0x17F6b2C738a63a8D3A113a228cfd0b373244633D), and [ATC](https://app.safe.global/settings/setup?safe=eth:0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956)), as previously [requested on the forum](https://research.lido.fi/t/lido-v2-may-1-2023-december-31-2023-lido-ongoing-grant-request/4476/11). Items 9-11.
+"""
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> bool | list[int | TransactionReceipt | None]:
+    """Prepare and run voting."""
+
+    rcc_multisig_address = "0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437"
+    pml_multisig_address = "0x17F6b2C738a63a8D3A113a228cfd0b373244633D"
+    atc_multisig_address = "0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956"
+
+    NO_registry = interface.NodeOperatorsRegistry(contracts.node_operators_registry)
+    prysmatic_labs_node_id = 27
+    prysmatic_labs_node_new_name = "Prysm Team at Offchain Labs"
+
+    call_script_items = [
+        # I. stETH transfers to RCC PML ATC
+        # 1. Transfer TBA stETH to RCC 0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437
+        make_steth_payout(
+            target_address=rcc_multisig_address,
+            steth_in_wei=1 * (10**18),
+            reference="Fund RCC multisig"
+        ),
+        # 2. Transfer TBA stETH to PML 0x17F6b2C738a63a8D3A113a228cfd0b373244633D
+        make_steth_payout(
+            target_address=pml_multisig_address,
+            steth_in_wei=1 * (10**18),
+            reference="Fund PML multisig"
+        ),
+        # 3. Transfer TBA stETH to ATC 0x9B1cebF7616f2BC73b47D226f90b01a7c9F86956
+        make_steth_payout(
+            target_address=atc_multisig_address,
+            steth_in_wei=1 * (10**18),
+            reference="Fund ATC multisig"
+        ),
+        # II. Change the on-chain name of node operator with id 27 from 'Prysmatic Labs' to 'Prysm Team at Offchain Labs'
+        # 4. Grant NodeOperatorsRegistry.MANAGE_NODE_OPERATOR_ROLE to voting
+        encode_permission_grant(
```

Comment:
<img width="482" alt="Screenshot 2023-10-31 at 14 22 17" src="https://github.com/lidofinance/scripts/assets/870356/9a5fd5f6-0c7b-42a6-8875-f269b178d1e3">
Agent already have MANAGE_NODE_OPERATOR_ROLE on Node Operators registry, so you have only to call set name method with agent forward method